app/Services/BaseService.php

```diff
@@ -155,6 +168,6 @@ protected function buildParameters(): array
 
     protected function containerName(): string
     {
-        return 'TO--' . $this->shortName() . '--' . $this->tag;
+        return 'TO--' . $this->shortName() . '--' . $this->nickname() . '--' . $this->tag;
```

Comment:
Can we put the nickname at the end and only add the `--` if there's a nickname?

Reply:
For sure! Remove the default unique string too?

Reply:
@nehero Oh, I hadn't even noticed that, actually!

I think... Let me think about it a bit. :)